utils/src/errors.rs

```diff
@@ -28,9 +27,9 @@ pub enum ServalError {
     #[error("the WASM executable terminated abnormally; code={}", result.code)]
     AbnormalWasmExit { result: WasmResult },
 
-    /// The WASMtime engine responded with an error.
-    #[error("wasmtime engine error")]
-    WasmEngineError(#[from] wasi_common::Error),
+    // A conversion for anyhow::Error
+    #[error("an anyhow! error")]
```

Comment:
One thing I don't love about the way that this stuff works is that the original error gets lost—if you call `to_string` on an instance of `ServalError::AnyhowError`, you just end up with `"an anyhow! error"`, which is...less useful than I would've hoped.